packages/@okta/vuepress-site/docs/guides/build-sso-integration/specify-your-settings/openidconnect/settings.md

```diff
@@ -0,0 +1,83 @@
+### Specify OIDC settings
+
+* On the **General** tab, in the **Application** area, you can rename your integration and select which [grant type options](https://developer.okta.com/blog/2017/07/25/oidc-primer-part-1#whats-a-response-type) are allowed.
+
+* If you only want to support direct SSO to your application (i.e. the integration is not launched from the Okta End-User Dashboard), then:
```

Comment:
avoid latin abbreviations such as i.e., use for example instead
conversational tone: is not -> isn't